.github/workflow_templates/build_push_exporters/action.yml

```diff
@@ -0,0 +1,53 @@
+name: build_push_exporters
+
+inputs:
+  image: 
+    required: true
+  builder_image:
+    required: true
+  s2i_log_level:
+    required: true
+  quay_imageregistry:
+    required: true
+  quay_imagenamespace:
+    required: true
+  quay_username:
+    required: true
+  quay_password:
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Set Image version
+      shell: bash
+      run: |
+        IMG_VER=$(grep quay.io charts/pelorus/subcharts/exporters/templates/_imagestream_from_image.yaml | grep -oP '(?<=default ")[^"]+')
```

Comment:
why get the version from this file and not `charts/pelorus/Chart.yaml`?

Reply:
and this version isn't the same for every exporter? why get it every time instead of just getting it before the jobs?

Reply:
The versions should be the same, but we reference the quay image tag in the `_imagestream_from_image.yaml` so I want to make this file source of truth not the Chart.yaml. Imagine one will bump the Chart.yaml without `_imagestream_from_image.yaml` which should not happen but it's possible, this way the tag will be wrong.